addons/sale_project_account/models/project.py

```diff
@@ -35,13 +35,14 @@ def action_open_project_vendor_bills(self):
 
     def _get_stat_buttons(self):
         buttons = super(Project, self)._get_stat_buttons()
-        buttons.append({
-            'icon': 'pencil-square-o',
-            'text': _('Vendor Bills'),
-            'number': self.vendor_bill_count,
-            'action_type': 'object',
-            'action': 'action_open_project_vendor_bills',
-            'show': self.user_has_groups('account.group_account_readonly') and self.vendor_bill_count > 0,
-            'sequence': 14,
-        })
+        if self.user_has_groups('account.group_account_readonly'):
+            buttons.append({
+                'icon': 'pencil-square-o',
+                'text': _('Vendor Bills'),
+                'number': self.vendor_bill_count,
+                'action_type': 'object',
+                'action': 'action_open_project_vendor_bills',
+                'show': self.user_has_groups('account.group_account_readonly') and self.vendor_bill_count > 0,
```

Comment:
Since we are sure the user has the group, you can directly do this:
```suggestion
                'show': self.vendor_bill_count > 0,
```